tests/unit/test_charm.py

```diff
@@ -15,36 +16,89 @@
 
 logger = logging.getLogger(__name__)
 
+DATABASE_URL = "http://6.5.6.5"
+DATABASE_USERNAME = "rock"
+DATABASE_PASSWORD = "paper"
+DB_APPLICATION_NAME = "mongodb-k8s"
+DB_RELATION_NAME = "database"
+NRF_APPLICATION_NAME = "nrf"
+NRF_RELATION_NAME = "fiveg_nrf"
+TLS_APPLICATION_NAME = "tls-certificates-operator"
```

Comment:
```suggestion
```

Reply:
Used this variable under def certificates_relation_id method.